modules/node_modules/@ncigdc/components/Case.js

```diff
@@ -156,8 +168,16 @@ const Case = ({
     });
   }, []);
 
+  const filesToAdd = files.filter(f => !cartFiles.some(cf => cf.file_id === f.file_id));
+
   return (
     <Column spacing={theme.spacing}>
+      <Row style={{ justifyContent: 'flex-end' }}>
+        <Button onClick={() => dispatch(toggleFilesInCart(filesToAdd.length ? filesToAdd : files))}>
+          {filesToAdd.length ? 'Add all files to the cart' : 'Remove all files from the cart'}
+        </Button>
```

Comment:
Is it possible for this to be a problem if "Add all files to cart" contains files that were already in the cart (e.g. individually added, added from a different cases, etc)?